tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/ResponseHandlerTests.cs

```diff
@@ -38,5 +42,188 @@ public async Task HandleUserResponse()
             var headers = (JObject)(user.AdditionalData["responseHeaders"]);
             Assert.Equal("value", (string)headers["test"][0]);
         }
+
```

Comment:
I think it would be great if we could have a scenario that has ```@removed": {"reason": "changed"}```
in the payload to validate/illustrate when an object is in the delta due to deletion.

Reply:
I agree with this, @MIchaelMainer.

Reply:
Otherwise, it LGTM.

Reply:
Addressed in 3d2e1d12